services/storage-users/pkg/config/config.go

```diff
@@ -94,7 +94,7 @@ type Drivers struct {
 
 // AsyncPropagatorOptions configures the async propagator
 type AsyncPropagatorOptions struct {
-	PropagationDelay time.Duration `yaml:"propagation_delay" env:"STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY" desc:"The delay in seconds between a change made to a tree and the propagation start on treesize and treetime. Multiple propagations are computed to a single one."`
+	PropagationDelay time.Duration `yaml:"propagation_delay" env:"OCIS_STORAGE_USERS_ASYNC_PROPAGATOR_PROPAGATION_DELAY" desc:"The delay in seconds between a change made to a tree and the propagation start on treesize and treetime. Multiple propagations are computed to a single one."`
```

Comment:
Not needed. See https://github.com/owncloud/ocis/issues/7292#issuecomment-1722875657 please revert

Reply:
☑️